### ISSUE TITLE
Problems with Centos 6.6 Install

### ISSUE THREAD

**Issue by dBlairMorton** [HazyResearch/deepdive#611]
I am trying to install DeepDive on a Centos 6.6 CLOSED system (i.e. no Internet).  I've pulled the deepdive-master.zip file over and tried to build it, but I keep getting an error that there is no git available.  I suspect I'm going to have to build it on an open system and pull the binary over, but I was wondering if anyone has successfully built DeepDive like this.

Thanks, Blair Morton

**Comment by alldefector** [comment]
@dBlairMorton yes, the build process does require internet access as it fetches code and dependencies. Building on an open system and then copying the binaries should work.

**Comment by EadrenKing** [comment]
DO NOT download zip file. Use git clone instead!
